src/Trace/Trace.php

```diff
@@ -46,36 +54,39 @@ class Trace implements \Serializable
     private $traceId;
 
     /**
-     * @var TraceSpan[] List of TraceSpans to report
+     * @var Span[] List of Span to report
      */
     private $spans = [];
 
     /**
      * Instantiate a new Trace instance.
      *
-     * @param ConnectionInterface $connection The connection to Stackdriver Trace.
      * @param string $projectId The id of the project this trace belongs to.
      * @param string $traceId [optional] The id of the trace. If not provided, one will be generated
      *        automatically for you.
-     * @param array $spans [optional] Array of TraceSpan constructor arguments. See
-     *        {@see Google\Cloud\Trace\TraceSpan::__construct()} for configuration details.
+     * @param string[] $spans [optional] Array of Span constructor arguments. See
```

Comment:
It seems `string` may not be correct, I believe this accepts an array of arrays.